### PR TITLE
ДЗ 1

### PR DESCRIPTION
**Pull request by aryabenkiy**
поехали. найди 7 отличий : ))

### CODE REVIEW

**Comment by aryabenkiy** [comment]
Pull requests создан через веб морду : ( к сожелению команд

File: HW1/SoftServeWOstl/FileWithStructur.cpp [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+            if (i==0) //Велика літера
+                {        
+                LastName.push_back(BigSymvol[(rand()%29 - 1)]);
+                }
+            else if (i<=LenghName) 
+                LastName.push_back(SmalSymvol[(rand()%33 - 1)]);
+         }
+        std::cout  << LastName << " \n";
+        //std::strncpy(WriteName, LastName, LenghName);
+        //WriteName[LenghName] = '\0';
```
Comment (DmytroHaponov) [inline HW1/SoftServeWOstl/FileWithStructur.cpp]: your commits must virtually never contain commented code. It's the first thing reviewer checks

File: HW1/SoftServeWOstl/FileWithStructur.cpp [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+    NameFile.close();                     // закриваемо файл
+    return 0;
+}
+
+
+bool CompareString(const char * s1, const char * s2)
+{
+    for(int i =0;i < strlen(s1); i++)
+    {
+        if (s1[i] < s2[i]) return false;
```
Comment (DmytroHaponov) [inline HW1/SoftServeWOstl/FileWithStructur.cpp]: is this the way to avoid putting "{}" ?)

File: HW1/SoftServeWOstl/FileWithStructur.cpp [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+
+    //для универсальності задамо питання кількості слів
+    int LenghFile = 0,i = 0,LenghName = 3;
+    std::string LastName;
+    
+    std::cout << "Скільки імен згенерувати?:\n";
+    std::cin>>LenghFile;    
+    std::srand ( time(NULL) );
+    
+    if (LenghFile < 1) return 0 ;
```
Comment (DmytroHaponov) [inline HW1/SoftServeWOstl/FileWithStructur.cpp]: Please don't get used to it

File: HW1/SoftServeWOstl/FileWithStructur.cpp [hunk 4]
```diff
@@ -0,0 +1,110 @@
+#include <cstdlib>
+#include <cstring>
+#include <fstream>
+#include <iostream>
+#include <string>
+
+namespace generator
+{
+    char BigSymvol[] = "АБВГҐДЕЄЖЗІКЛМНОПРСТУФХЦЧШЩЮЯ";
+    char SmalSymvol[] = "fбвгґдеєжзиіїйклмнопрстуфхцчшщьюя";
```
Comment (DmytroHaponov) [inline HW1/SoftServeWOstl/FileWithStructur.cpp]: "f" instead of "а" )

File: HW1/SoftServeWOstl/FileWithStructur.cpp [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+        NameFile.getline(buff2, 11);
+        if (CompareString(buff1,buff2)) 
+        {
+            NameFileTemp << buff1 << "\n";
+            for (int i=0; i<=10; i++)
+            buff1[i] = buff2[i];
+        }
+        else NameFileTemp << buff2 << "\n";   
+        //getline(NameFile,buff1);
+        std::cout << buff1<<"\n";
```
Comment (DmytroHaponov) [inline HW1/SoftServeWOstl/FileWithStructur.cpp]: let's get used to this "always use {} after if, for, else, etc"

File: HW1/SoftServeWOstl/FileWithStructur.cpp [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+    }
+    
+    return true;
+}
+
+int SortFile(int argc, char** argv) 
+{
+    
+    setlocale(LC_ALL, ".1251");
+    char buff1[11],buff2[11];
```
Comment (DmytroHaponov) [inline HW1/SoftServeWOstl/FileWithStructur.cpp]: what are these magic numbers?

**Pull request merged**